plugins/org.jkiss.dbeaver.core/src/org/jkiss/dbeaver/ui/preferences/PrefPageDatabaseUserInterface.java

```diff
@@ -74,11 +77,13 @@ public class PrefPageDatabaseUserInterface extends AbstractPrefPage implements I
 
     @Nullable
     private Combo clientTimezone;
+    private Combo cmbDefaultEncoding;
```

Comment:
```suggestion
    private Combo defaultEncodingCombo;
```

We don't really use abbreviations such as cmb, btn, etc.